dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/DisposedOpenTelemetryInfo.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal.common.info;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+
+/**
+ * An impl of OpenTelemetryInfo that only occurs if an OpenTelemetryInfo was created after the relevant application has shut down.
+ */
+public class DisposedOpenTelemetryInfo implements OpenTelemetryInfo {
+
+    private static final TraceComponent tc = Tr.register(DisposedOpenTelemetryInfo.class);
+
+    private String appName = "Unkown";
+
+    public DisposedOpenTelemetryInfo() {}
+
+    public DisposedOpenTelemetryInfo(String appName) {
+        this.appName = appName;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean getEnabled() {
+        return false;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public OpenTelemetry getOpenTelemetry() {
+        logWarning();
+        return OpenTelemetry.noop();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Tracer getTracer() {
+        logWarning();
+        return OpenTelemetry.noop().getTracerProvider().get("");
+    }
```

Comment:
These methods could potentially spam out warnings which doesn't seem particularly helpful. I'm still thinking more about how I think this should work.